internal/zinc-classfile/src/main/scala/sbt/internal/inc/classfile/JavaAnalyze.scala

```diff
@@ -192,7 +192,7 @@ private[sbt] object JavaAnalyze {
 
       def readInheritanceDependencies(classes: Seq[Class[_]]) = {
         val api = readAPI(source, classes)
-        api.groupBy(_._1).mapValues(_.map(_._2))
+        api.groupBy(_._1).iterator.map { case (k, v) => k -> v.map(_._2) }
```

Comment:
```suggestion
        // avoid .mapValues(...) because of its viewness (scala/bug#10919)
        api.groupBy(_._1).iterator.map { case (k, v) => k -> v.map(_._2) }
```

Reply:
https://github.com/sbt/zinc/compare/0f2304ee2d4ab23e7038988782a90eef403c3f53..a1492e6f2a7529a41d98359277c8213fd3168fc1